src/taskgraph/transforms/docker_image.py

```diff
@@ -5,18 +5,15 @@
 import logging
 import os
 import re
-from textwrap import dedent
-
-from voluptuous import Optional, Required
+from typing import Dict, List, Optional
 
 import taskgraph
 from taskgraph.transforms.base import TransformSequence
+from taskgraph.transforms.task import TaskDescriptionIndexSchema
 from taskgraph.util import json
 from taskgraph.util.docker import create_context_tar, generate_context_hash
 from taskgraph.util.schema import Schema
 
-from .task import task_description_schema
```

Comment:
Same here, anytime we see a schema being referenced in another file, we should continue referencing the new msgstruct style schema from the same file.